### DISCUSSION TITLE
Uncategorized list

### DISCUSSION THREAD

**Discussion by totpero** [community/community#107581] [Other Feature Feedback, Questions, & Ideas] [answered]
Add feature all stared repo that is not in any list to add into uncategorized list.
This help to add this into specific list.

**Comment by Termar2672** [comment]
My Facebook account got hacked.  All my credentials opens another account. The profile doesn't show Hardly shows anything.  No pic, no info. It does her pics a d her posts
Can you get mine back without losing everything I have. I need that specific account

**Comment by yaSebastian** [comment]
+1

This would be really important to sort the long list of starred repositories some of us might have amassed over the years.

**Reply by totpero** [reply, marked as answer]
This is exactly the problem I have now. I have 2600 repositories stars and many stars is before this nice feature list was added.

**Comment by Crystal-RainSlide** [comment]
This Uncategorized list thing would be more convenient, I'd prefer this.

Otherwize, at least there shall be a way to filter those uncategorized repos, so we can work on them.

**Comment by Cjuan4631** [comment]
Okta

**Comment by Hasmuni5757** [comment]
**[]()**

**Reply by Hasmuni5757** [reply]
Go

**Comment by S0me1-dev** [comment]
extra needed one function!

**Comment by tauniaxd** [comment]
Greatly needed funtion!
